manifests/f/FreedomScientific/FreedomScientificJAWS2022/2022.2202.38.400/FreedomScientific.FreedomScientificJAWS2022.installer.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: FreedomScientific.FreedomScientificJAWS2022

```

The Identifier could be cleaned up a bit;  Remember to adjust the folder structure to `/manifests/f/FreedomScientific/Jaws/2022/2022.2202.38.400/` if changing this value
```suggestion
PackageIdentifier: FreedomScientific.JAWS.2022
```